package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Docs for SolidStart",
   "type": "module",
   "scripts": {
+    "compile": "tsc --noEmit",
```

Comment:
I generally prefer putting all flags in `tsconfig.json`, so running `tsc` on the CLI is the same as whatever compile/build command is used. Is there anything stopping this PR from adding `"noEmit": true` in `tsconfig.json`'s `compilerOptions`?